src/omni-slider.js

```diff
@@ -1,532 +1,540 @@
-/* Constructor function
- * elementContainer - this acts as a wrapper for the slider, all of the sliders DOM elements will be transcluded inside the <element> provided
- * options - contains the options for the slider
- */
-function Slider(elementContainer, options) {
+(function() {
```

Comment:
This would be necessary on it's own, but Babel is wrapping the entire piece in an IIFE with `use strict` on build as it is.